### PR TITLE
L0 global majority snapshot endpoints

### PR DESCRIPTION
**Pull request by TheMMaciek**

### CODE REVIEW

File: modules/core/src/main/scala/org/tessellation/http/routes/GlobalSnapshotRoutes.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+
+final case class GlobalSnapshotRoutes[F[_]: Monad](globalSnapshotStorage: GlobalSnapshotStorage[F])
+    extends Http4sDsl[F] {
+  private val prefixPath = "/global-snapshot"
+
+  private val httpRoutes: HttpRoutes[F] = HttpRoutes.of[F] {
+    case GET -> Root / "latest" =>
+      Ok(globalSnapshotStorage.getLast)
```
Comment (marcinwadon) [inline modules/core/src/main/scala/org/tessellation/http/routes/GlobalSnapshotRoutes.scala:19]: To be consistent maybe it should be either "/latest" and ".getLatest" or "/last" and ".getLast"?

File: modules/core/src/main/scala/org/tessellation/http/routes/GlobalSnapshotRoutes.scala [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    case GET -> Root / "ordinal" / SnapshotOrdinalVar(ordinal) =>
```
[the remark this answers is not in the document]
Comment (TheMMaciek) [reply modules/core/src/main/scala/org/tessellation/http/routes/GlobalSnapshotRoutes.scala]: Yeah, we can do it like this. I adjusted the code. We don't have the functionality for fetching snapshot by hash in the storage yet, so I'm not adding that endpoint currently.

**Review by marcinwadon — APPROVED** [review]
Please make sure on the cluster both endpoints are working correctly (don't interfere each other)

**Comment by TheMMaciek** [comment]
> 

Already tested, no clash happening.

**Pull request merged**